tests/scale/scale_test.go

```diff
@@ -0,0 +1,45 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+func TestBasic(t *testing.T) {
```

Comment:
we have the same name 'TestBasic' in the basic tests
doesn't go conflict when two methods with same name are defined in the same package?